ner_v2/detectors/temporal/time/en/data/timezone_variations.csv

```diff
@@ -0,0 +1,10 @@
+timezone_value,timezone_variants
+IST,IST|Indian Time|Indian Standard Time
+EST,EST|Eastern Standard Time|Eastern Time|ET|EDT
+CST,CST|Central Standard Time|Central Time|CT|CDT
+MST,MST|Mountain Standard Time|Mountain Time|MT|MDT
+PST,PST|Pacific Standard Time|Pacific Time|PT|PDT
+AKST,AKST|Alaska Standard Time|Alaska Time|AKDT
+HST,HST|Hawaii Standard Time|HDT
+HAST,HAST|Hawaii-Aleutian Standard Time|Hawaii Aleutian Standard Time|Hawaii Time|HADT
```

Comment:
can we combine timezone_variations and timezone files?

Reply:
sure